src/platform/monitoring/DowntimeNotification/actions/index.js

```diff
@@ -50,18 +55,42 @@ export function dismissDowntimeWarning(appTitle) {
 }
 
 export function getScheduledDowntime() {
-  return async dispatch => {
+  return async (dispatch, state) => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
     let data;
+
+    // create global downtime data if feature toggle is enabled and if
+    // current time is in downtime window
+    // default to empty array
+    const { downtimeStart, downtimeEnd } = scheduledDowntimeWindow;
+    const globalDowntimeData =
+      (toggleValues(state).vaGlobalDowntimeNotification &&
+        moment().isAfter(downtimeStart) &&
+        moment().isBefore(downtimeEnd) &&
+        createGlobalMaintenanceWindow({
+          startTime: downtimeStart,
+          endTime: downtimeEnd,
+        })) ||
+      [];
```

Comment:
these changes impact the DowntimeNotification component. Basically this generates the response data that the component would get from the api if _all of the services are down_ and adds it to a failure action.